### DISCUSSION TITLE
Clarification needed regarding stopping K3s

### DISCUSSION THREAD

**Discussion by sylvainfaivre** [k3s-io/k3s#8382] [General]
Hello,

I have questions about this documentation page : https://docs.k3s.io/upgrades/killall 

Firstly, the page mentions « To allow high availability during upgrades, the K3s containers continue running when the K3s service is stopped. »
But this is less visible than the part about the killall script, and this is the only page about stopping k3s. To be honest, after reading this page I didn't remember the part about being able to stop only k3s and keep the containers running.
I believe this could be clearer with code blocks giving the instructions for stopping the service, similar to this paragraph : https://docs.k3s.io/upgrades/manual#restarting-k3s

Secondly, this page is not explicit about which usecases need you to run the killall script.
The "Manual Upgrades" page gives instructions for stopping and restarting the k3s binary or the k3s service, while the "Automated Upgrades" page doesn't mention restarting k3s (which I suppose is not needed, since it's automated).
So it seems that the killall script is not needed in the upgrade process, even tought it is the first page in the Upgrades section. I could be wrong about this, since I'm just in the process of getting to know k3s.

**Comment by brandond** [comment]
What is the question? When do you need to run the killall script? You only need to run it if you for some reason want to terminate all the pods. You don't need to do it when restarting or upgrading k3s.

If you have suggestions on how to improve the docs, an issue or PR to https://github.com/k3s-io/docs would be appreciated!

**Reply by sylvainfaivre** [reply]
You're right, the only real question was the one you answered, thank you.

I don't think I know enough about k3s to make the docs PR, but I'm going to create an issue.
